From a69e8e46dfdb64cba590427114db8bbdeec4b708 Mon Sep 17 00:00:00 2001
From: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Date: Fri, 29 Sep 2023 14:31:10 +0200
Subject: [PATCH] arm64: defconfig: enable USB_HIDDEV in addons fragment

Enable devices with a separate event interface on /dev/usb/hiddevX

Signed-off-by: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Change-Id: I2fb619747c66329d25d7375e34d3540d9daf6686
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -2,3 +2,4 @@
 CONFIG_DYNAMIC_DEBUG=y
 CONFIG_OF_OVERLAY=y
 CONFIG_REGULATOR_ARM_SCMI=y
+CONFIG_USB_HIDDEV=y
